INPUT_SCHEMA.json

```diff
@@ -31,6 +31,30 @@
             "editor": "requestListSources",
             "sectionCaption": "Direct URLs"
         },
+        "videosFromDate": {
+            "title": "Videos from",
+            "type": "string",
+            "editor": "textfield",
+            "description": "Scrape the videos from the date e.g <br />30 minutes <br />1 week <br />5 months <br />4 years, etc.<br />for 'Earliest' empty the field",
+            "pattern": "(^(1|([^0a-zA-Z ][0-9]{0,3})) (minute|hour|day|week|month|year))s?",
+            "minLength": 0,
+            "maxLength": 16,
+            "prefill": "01-01-2000",
```

Comment:
Where is this format coming in? Standard is to use ISO format, e.g. YYYY-MM-DD so I would use that if we can

Reply:
the pattern should actually be like this https://github.com/gippy/instagram-scraper/blob/726043ae1eaef7af4a524932a1db18b33b35d383/INPUT_SCHEMA.json#L92

omit the `prefill`